doc/README.open_ce_build.md

```diff
@@ -37,28 +37,27 @@ directory; simply specify the file for whichever package environment you want.
 > you may do so if you wish to perform an individual build of your own
 > for any given Open-CE feedstock repository.
 
-### Docker build
+### Container build
 
-The `--docker_build` option will build an image and run the build command
-inside of a container based on the new image.
+The `--container_build` option will build an image and run the build command
+inside of a container based on the new image. The `--container_tool` option can
+be passed to specify container tool to be used. 
 
-Along with `--docker_build` option, `--docker_build_args` can be passed
-to set docker build options like environment variables or other settings
+Along with `--container_build` option, `--container_build_args` can be passed
+to set container build options like environment variables or other settings
 like cpusets.
 
 ```shell
-    ./open-ce/open_ce/open-ce build env --docker_build --docker_build_args="--build-arg ENV1=test1 --cpuset-cpus 0,1" open-ce-environments/envs/pytorch-env.yaml
+    ./open-ce/open_ce/open-ce build env --container_build --container_tool podman --docker_build_args="--build-arg ENV1=test1 --cpuset-cpus 0,1" open-ce-environments/envs/pytorch-env.yaml
```

Comment:
--docker_build_args to be changed to `--container_build_args`

Reply:
Yes, am verifying all merge changes. Thanks will change.